src/site/_includes/macros/postcard.njk

```diff
@@ -0,0 +1,44 @@
+{% macro postcard(collectionItem, className = '', featured=false) %}
+
+{% set url = collectionItem.url %}
+{% set data = collectionItem.data %}
+{% set thumbnail = data.thumbnail or data.hero or null %}
+{% set alt = data.alt or '' %}
+{% set isDraft = 'w-card--draft' if data.draft %}
+{% set authors = data.authors or [] %}
+
+<div class="w-card {{className}} {{isDraft}}" role="listitem">
+  <article class="w-card-base {{ 'w-card-base--featured' if featured }}">
+    {% if thumbnail %}
+      <div class="w-card-base__cover w-card-base__cover--with-image }}">
+        <a href="{{url}}" class="w-card-base__link" tabindex="-1" aria-hidden="true">
```

Comment:
Would you add a comment here saying that we intentionally hide the thumbnail image from screen readers because it appears out of context from the rest of the article content and we suspect it might be confusing. Since the link for the article is duplicated in the heading down below, we think this might be an OK tradeoff.